config/locales/views/users/nb.yml

```
@@ -7,9 +7,14 @@ nb:
-
+    reset:
+      success: Kontoen din har blitt tilbakestilt. Data vil bli slettet i bakgrunnen om en stund.
+      unauthorized: Du er ikke autorisert til å utføre denne handlingen.
+    reset_with_sample_data:
+      success: Kontoen din har blitt tilbakestilt og demodata forberedes. Du vil snart se eksempeldata.
+    menu_settings: Innstillinger
+    menu_changelog: Endringslogg
+    menu_feedback: Tilbakemelding
+    menu_contact: Kontakt
+    menu_logout: Logg ut
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)